### PR TITLE
RDM-12741: AC FTA - Create MVP Scenarios for get Case

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-12741

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/aat/resources/features/F-201 - New Access Control Get Case/F-201_CT1.td.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+  "_guid_": "F-201_CT1",
+
+  "productName": "CCD Data Store",
```
Comment (noronhaa) [inline src/aat/resources/features/F-201 - New Access Control Get Case/F-201_CT1.td.json:4]: for all files using the /case-types/{ctid}/cases endpoint lets create and extend a base file like F-201_GetCaseBase.td.json and put the common lines of code here like the api details and request headers, then we can remove this common lines from all the files extending them

File: src/aat/resources/features/F-201 - New Access Control Get Case/F-201_CT1.td.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  "specs": [
+    "a role assignment to access CaseType CT1",
+    "attempts to create a case for CT1",
+    "to create a full case"
+  ],
+
+  "users" : {
+    "invokingUser": {
+      "username": "ccd.ac.superuser@gmail.com",
+      "password": "Pa55word11"
```
Comment (noronhaa) [inline src/aat/resources/features/F-201 - New Access Control Get Case/F-201_CT1.td.json:18]: lets hide all of the passwords with [[$CCD_BEFTA_CASEWORKER_1_PWD]] for now

File: src/aat/resources/features/F-201 - New Access Control Get Case/Grant_Role_Assignment.td.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
+{
+  "_guid_" : "GRANT_CASE_ROLE_ASSIGNMENT",
```
Comment (noronhaa) [inline src/aat/resources/features/F-201 - New Access Control Get Case/Grant_Role_Assignment.td.json:2]: for all files using the /am/role-assignments  endpoint lets create and extend a base file like F-201_GrantRoleAssignment.td.json and put the common lines of code here like the api details and request headers, then we can remove this common lines from all the files extending them

**Comment by Thor-tech-of-metal** [comment]
Kiran  you will need to rebase on to of RDM-12732 .  This is ashley pr , I mergend my pr there . RDM-12732

File: src/aat/resources/features/F-201 - New Access Control Get Case/Grant_Role_Assignment.td.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+  "productName": "Role Assignment Service",
+  "operationName": "post role assignment",
+  "method": "POST",
+  "uri": "http://localhost:4096/am/role-assignments",
```
Comment (noronhaa) [inline src/aat/resources/features/F-201 - New Access Control Get Case/Grant_Role_Assignment.td.json:7]: we need to make this host dynamic or env var  as this wont work i higher up envs. we can use ROLE_ASSIGNMENT_HOST env var

File: src/aat/resources/features/F-201 - New Access Control Get Case/Grant_Role_Assignment.td.json (hunk 3 above)
Comment (noronhaa) [inline src/aat/resources/features/F-201 - New Access Control Get Case/Grant_Role_Assignment.td.json:2]: Lets prefix the file name and guid with F201_ as these files are unique to this set of tests

**Review by noronhaa — CHANGES_REQUESTED** [review]
- base file for get case endpoint files
- base file for grant role assignment files
-  prefix grant role assignment files with feature file tag/number

**Review by Thor-tech-of-metal — COMMENTED** [review]
I also noticed the need to add a base json files.

**Review by Thor-tech-of-metal — APPROVED** [review]

**Pull request closed without merging**